docs/2.7.0/docs/app-dev/explicit-contract-disclosure.rst

```diff
@@ -0,0 +1,261 @@
+.. Copyright (c) 2023 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+.. _explicit-contract-disclosure:
+
+Explicit Contract Disclosure (Experimental)
+###########################################
+
+Canton 2.7 introduces explicit contract disclosure as a feature
+that allows you to seamlessly delegate contract read rights to a non-stakeholder using off-ledger data distribution.
```

Comment:
```suggestion
that allows you to seamlessly delegate contract read rights to a non-stakeholder by sending the essential metadata off-ledger.
```
maybe ? anw now that I read all the section it makes sense what it means. I just thought that data distribution is already a mechanism when I read the sentence for the first time.

Reply:
I am afraid of too much repetition with this essential metadata